fs/semaphore/sem_open.c

```diff
@@ -73,6 +73,7 @@
  *        create the semaphore.
  *     oflags = O_CREAT|O_EXCL:  Create a new semaphore
  *        unless one of this name already exists.
+ *   ecode - Location to return the errno value in the event of an error
```

Comment:
Maybe `in the event of an error` -> `in case of an error`?

Reply:
> case of an error

done.

Reply:
if so, let's change to:
```
int nxsem_open(FAR sem_t **sem, FAR const char *name, int oflags, ...)
```
which is closer to sem_open prototype

Reply:
> if so, let's change to:
> 
> ```
> int nxsem_open(FAR sem_t **sem, FAR const char *name, int oflags, ...)
> ```
> 
> which is closer to sem_open prototype

done.